picard/ui/metadatabox.py

```diff
@@ -240,7 +277,14 @@ def contextMenuEvent(self, event):
                 menu.addAction(edit_tag_action)
             removals = []
             useorigs = []
+            item = self.currentItem()
             for tag in tags:
+                if tag in LOOKUP_TAGS:
+                    if item.column() == 1 or item.column() == 2:
+                        if len(tags) == 1:
```

Comment:
You also need to make sure that the tag is not empty. That can happen, for example, if there's no old value for a tag.